applications/build-modular-monolith-with-platformatic/services/media-service/platformatic.composer.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://platformatic.dev/schemas/v0.27.0/composer",
+  "server": {
+    "hostname": "{PLT_SERVER_HOSTNAME}",
+    "port": "{PORT}",
+    "logger": {
+      "level": "{PLT_SERVER_LOGGER_LEVEL}"
+    }
+  },
+  "composer": {
+    "services": [
+      {
+        "id": "books-service",
+        "openapi": {
+          "url": "/documentation/json",
+          "ignore": [
+            { "path": "/books/", "methods": ["POST", "PUT", "DELETE"] }
```

Comment:
Blocking issue: I can't ignore the `/books/{id}` path. I receive an error due to the curly braces, which cause it to be interpreted as a environment variable placeholder.

Related issue: https://github.com/platformatic/platformatic/issues/1116

Reply:
@ivan-tymoshenko could you take a look?